proto/repositories.bzl

```diff
@@ -14,22 +14,24 @@
 
 """Dependencies and toolchains required to use rules_proto."""
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@bazel_tools//tools/build_defs/repo:java.bzl", "java_import_external")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", _http_archive = "http_archive")
 load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
-load("//proto/private:dependencies.bzl", "dependencies", "maven_dependencies", "protobuf_workspace")
+
+def http_archive(name, **kwargs):
```

Comment:
Please inline.

As much as I like a hack to redefine `http_archive`, this decreases readability. Using `maybe` in the code where the depo is imported is a clear signal to the reader. Besides this all other repos I'm familiar with have `maybe` used in-line. You're not saving much lines of code.